commerce_coordinator/apps/titan/tests/test_clients.py

```diff
@@ -42,7 +40,7 @@ class TitanPaymentClientMock(MagicMock):
     return_value = {
         'orderUuid': ORDER_UUID,
         'state': PaymentState.PROCESSING.value,
-        'responseCode': 'test-code',
+        'referenceNumber': 'test-code',
```

Comment:
Thank you for fixing the titan payment client mock. I noticed this attribute has been renamed in API spec but it was not reflected in code.